assets/translations/fr.json

```diff
@@ -142,6 +142,20 @@
     "change_pass": "Changer le mot de passe",
     "password_at_least_characters": "Le mot de passe doit être au moins de 6 caractères"
   },
+  "password_recovery_form": {
+    "password_at_least_characters": "Le mot de passe doit être au moins de 6 caractères",
```

Comment:
If it's easy: can this `6` be replaced with the constant we use when checking the password?